packages/ui-kit-svgr-template/index.js

```diff
@@ -32,4 +33,4 @@ const template = ({ imports, componentName, jsx, exports, interfaces }, { tpl })
   `;
 };
 
-export default template;
+module.exports = template;
```

Comment:
Curious - what made you have to switch module import/export notation here?

Reply:
Ah, I see this is a separate mini-package for the `svgr-template`, written in just `js`

Reply:
I didn't think it was necessary to introduce another build step for this template, so I wrote it in plain js. I think our current tsconfig won't accept js files from other packages with ESM syntax. I can try to change it, as I'll be messing with the tsconfigs anyway to fix the type issue the components package currently has.

Reply:
Actually, scratch that, I remember now. The storybooks internal tsconfig, which is not straightforward to change, won't accept js files with ESM syntax, so I can only use CJS.

Reply:
Got it - no need to change was mostly just curious.